common/flatpak-installation.c

```diff
@@ -1648,6 +1648,33 @@ flatpak_installation_get_config (FlatpakInstallation *self,
   return flatpak_dir_get_config (dir, key, error);
 }
 
+/**
+ * flatpak_installation_get_default_languages:
+ * @self: a #FlatpakInstallation
+ * @error: return location for a #GError
+ *
+ * Get the default languages being used by the user/system
+ * on the ostree configuration files, stored either by xa.languages
```

Comment:
This doesn't quite match the implementation. You could say "For user installations, this is determined from the `xa.languages` config key, or if unset the combination of `xa.default-languages` and languages set by environment variables. For system installations, if `xa.languages` is unset, the combination of `xa.default-languages` and the list returned by `AccountsService` is used."

Or just don't specify how it's implemented.